controllers/v1/userProjects.js

```diff
@@ -1125,5 +1126,83 @@ module.exports = class UserProjects extends Abstract {
                 }
         })
     }
+    /**
+    * @api {post} /improvement-project/api/v1/userProjects/listPendingProjects
+    * Lists projects based on the status 
+    * @apiVersion 1.0.0
+    * @apiGroup User Projects
+    * @apiSampleRequest /improvement-project/api/v1/userProjects/listPendingProjects
+    * @apiParamExample {json} Response:
+    /**
+
+    /**
+     * Lists projects based on the status
+     * @method
+     * @name listPendingProjects
+     * @returns {JSON} Project details
+    */
+
+    async listPendingProjects(req) {
+        return new Promise(async (resolve, reject) => {
+            
+                try {
+                    let listOfProjects = await projectsHelper.details({
+                        userId:req.userDetails.userInformation.userId,
+                        status:req.query.status
+                    },['title','description','_id','userId','isAPrivateProgram','createdBy','status','createdAt','deleted'])
+                    
+                    return resolve({
+                        message: 'success',
+                        result: listOfProjects
+                    });
+                    
+                } catch (error) {
+                    return reject({
+                        status: error.status || HTTP_STATUS_CODE.internal_server_error.status,
+                        message: error.message || HTTP_STATUS_CODE.internal_server_error.message,
+                        errorObject: error
+                    });
+                }
+        })
+    }
+    /**
+    * @api {post} /improvement-project/api/v1/userProjects/listCreatedProjects
+    * Lists projects created by the userId
+    * @apiVersion 1.0.0
+    * @apiGroup User Projects
+    * @apiSampleRequest /improvement-project/api/v1/userProjects/listCreatedProjects
+    * @apiParamExample {json} Response:
+    /**
+
+    /**
+     * Lists projects created by the userId
+     * @method
+     * @name listCreatedProjects
+     * @returns {JSON} Project details
+    */
+
+
+    async listCreatedProjects(req) {
+        return new Promise(async (resolve, reject) => {
```

Comment:
wen dont need this controller @borkarsaish65

Reply:
explaination given above